### DISCUSSION TITLE
overriding mode 0444 (r--r--r--)?

### DISCUSSION THREAD

**Discussion by amolvyas-6** [pystardust/ani-cli#1430] [Q&A]
Whenever i am playing an episode, the video is getting played but i am also getting this error message:
```
cp: cannot create regular file '/c/Users/<myUserName>/.local/state/ani-cli/ani-hsts.new': Permission denied
/c/Users/<myUserName>/scoop/apps/ani-cli/current/ani-cli: line 240: /c/Users/<myUserName>/.local/state/ani-cli/ani-hsts.new: Permission denied
mv: replace '/c/Users/<myUserName>/.local/state/ani-cli/ani-hsts', overriding mode 0444 (r--r--r--)?
```
BTW i am using windows 11 and have downloaded ani-cli with scoop.

**Comment by port19x** [comment]
That's strange. @ykhan21 might know more.

I suspect the history feature doesn't work for you given this output. Can you confirm?

**Reply by amolvyas-6** [reply]
yea the history wasn't working. But i anyways had to reset my PC. After resetting it, its working fine
